### PR TITLE
Spark: hdi deployment methods with livy

### PR DESCRIPTION
**Pull request by mamccrea**
## Summary

Add how-to that includes Apache Livy deployment method to HDI.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/spark/how-to-guides/hdinsight-deploy-methods.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+2. Copy the ssh login information and paste the login into a terminal. Sign in to your cluster using the password you set during cluster creation. You should see messages welcoming you to Ubuntu and Spark.
+
+3. Use the **spark-submit** command to run your app on your HDInsight cluster. Remember to replace **mycontainer** and **mystorageaccount** in the example script with the actual names of your blob container and storage account.
+
+   ```bash
+   $SPARK_HOME/bin/spark-submit \
+   --master yarn \
+   --class org.apache.spark.deploy.DotnetRunner \
+   wasbs://mycontainer@mystorageaccount.blob.core.windows.net/microsoft-spark-2.3.x-0.6.0.jar \
```
Comment (bamurtaugh) [inline docs/spark/how-to-guides/hdinsight-deploy-methods.md]: We may want to add a note telling users to adjust `2.3.x-0.6.0` based on their specific version of Spark and .NET for Apache Spark, i.e.:

Note: Replace `microsoft-spark-2.3.x-0.6.0.jar` with the appropriate jar file you're using for deployment. `2.3.x` represents the version of Apache Spark, and `0.6.0` represents the version of the [.NET for Apache Spark worker](https://github.com/dotnet/spark/releases).

File: docs/spark/how-to-guides/hdinsight-deploy-methods.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+title: Submit a .NET for Apache Spark job to Azure HDInsight
+description: Learn how to submit a .NET for Apache Spark job to Azure HDInsight using spark-submit and Apache Livy.
+ms.date: 11/14/2019
+ms.topic: conceptual
+ms.custom: mvc,how-to
+---
+
+# Submit a .NET for Apache Spark job to Azure HDInsight
+
+There are two ways to deploy your .NET for Apache Spark job to HDInsight: `spark-submit` and Apache Livy.
```
Comment (bamurtaugh) [inline docs/spark/how-to-guides/hdinsight-deploy-methods.md]: Should we add a note/link here asking users to view the HDI doc before this one?

File: docs/spark/how-to-guides/hdinsight-deploy-methods.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    "args":["abfss://<your-file-system-name>@<your-storage-account-name>.dfs.core.windows.net/<some dir>/<your app>.zip","<your app>","<app arg 1>","<app arg 2>,"...","<app arg n>"]
+}
+EOF
+```
+
+## Next steps
+
+* [Get started with .NET for Apache Spark](../tutorials/get-started.md)
+* [Deploy a .NET for Apache Spark application to Azure HDInsight](../tutorials/hdinsight-deployment.md)
+* [HDInsight Documentation](https://docs.microsoft.com/azure/hdinsight/)
```
Comment (Youssef1313) [inline docs/spark/how-to-guides/hdinsight-deploy-methods.md]: This fixes a lint error

```suggestion
* [HDInsight Documentation](https://docs.microsoft.com/azure/hdinsight/)

```

File: docs/spark/how-to-guides/hdinsight-deploy-methods.md (hunk 2 above)
Comment (mamccrea) [reply docs/spark/how-to-guides/hdinsight-deploy-methods.md]: I don't think it's necessary as a prereq, but it is linked in "next steps" as additional reading.

Comment (bamurtaugh) [reply docs/spark/how-to-guides/hdinsight-deploy-methods.md]: Okay makes sense!

**Comment by mamccrea** [comment]
@mairaw - can you please do final review for merge?

**Review by BillWagner — APPROVED** [review]
This LGTM @mamccrea 

You can :shipit: when ready.

**Pull request merged**